src/main/java/uk/gov/hmcts/reform/pbis/MessageProcessingResult.java

```diff
@@ -0,0 +1,89 @@
+package uk.gov.hmcts.reform.pbis;
+
+import java.util.Map;
+import java.util.Set;
+import java.util.stream.Collectors;
+import javax.validation.ConstraintViolation;
+import uk.gov.hmcts.reform.pbis.model.PrivateBetaRegistration;
+
+
+public class MessageProcessingResult {
+
+    public final MessageProcessingResultType resultType;
+    public final ProcessingError errorDetails;
+
+    public MessageProcessingResult(
+        MessageProcessingResultType resultType,
+        ProcessingError error
+    ) {
+        this.resultType = resultType;
+        this.errorDetails = error;
+    }
+
+    public static MessageProcessingResult success() {
+        return new MessageProcessingResult(MessageProcessingResultType.SUCCESS,null);
```

Comment:
Missing space after `,`.
Is checkstyle enabled on this project?

Reply:
Fixed. Yes, I've just made a line too long locally and build failed because of complaining checkstyle.